metrics/github/backfill.py

```diff
@@ -10,56 +9,152 @@
 from ..timescaledb import TimescaleDBWriter
 from ..timescaledb.tables import GitHubPullRequests
 from ..tools.dates import date_from_iso, iter_days, previous_weekday
-from .prs import process_prs
+from .api import session
+from .prs import drop_archived_prs, process_prs
 
 
 setup_logging()
 
 log = structlog.get_logger()
 
 
-def get_data(db, org):
-    subprocess.check_call(["github-to-sqlite", "repos", db, org])
-
-    con = sqlite3.connect(db)
-    cur = con.cursor()
+def get_query_page(*, query, session, cursor, **kwargs):
```

Comment:
You've introduced another complete set of functions for querying the GraphQL API. I think you might be about to remove the non-backfill codepath, in which case there's no need to worry about this duplication; but if that's not the case then please can you deduplicate?

Reply:
Yep, this is all going in the next PR!